pints/tests/test_toy_stochastic_logistic_model.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+#
+# Tests if the stochastic logistic growth (toy) model works.
+#
+# This file is part of PINTS.
```

Comment:
There's a new template for these headers now (which is why the "copyright" test is failing)